### PR TITLE
Update versions of dependencies

### PR DESCRIPTION
**Pull request by bkamins**
Hopefully this does not break anything ...

### CODE REVIEW

**Comment by bkamins** [comment]
All looks clean.

File: Project.toml [hunk 1]
```diff
@@ -16,11 +16,11 @@ SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
...
 [compat]
 ArnoldiMethod = "0.0.4"
-ColorTypes = "0.9"
-Colors = "0.11"
+ColorTypes = "0.9, 0.10"
+Colors = "0.11, 0.12"
 Compose = "0.7, 0.8"
-LightGraphs = "1.1"
-VisualRegressionTests = ">= 0.2"
+LightGraphs = "1.1, 1.2, 1.3"
```
Comment (matbesancon) [inline Project.toml:22]: this one should not be necessary? All 1.x versions should be fine greater that 1.1

Comment (bkamins) [reply Project.toml:22]: ah - this is a difference between pre and post 0.x release - right?

Comment (bkamins) [inline Project.toml:22]:
```suggestion
LightGraphs = "1.1"
```

Comment (matbesancon) [reply Project.toml]: yes exactly, otherwise this looks good, merging when it goes green

**Comment by bkamins** [comment]
Thank you!

Do you know who is managing bumping version of this package and tagging releases?

**Pull request merged**